metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-env.xml

```diff
@@ -253,69 +253,295 @@
         </value-attributes>
     </property>
     <property>
-        <name>global-json</name>
-        <display-name>global.json template</display-name>
-        <description>This is the jinja template for global.json file</description>
-        <value>
-{
-"es.clustername": "{{ es_cluster_name }}",
-"es.ip": "{{ es_url }}",
-"es.date.format": "yyyy.MM.dd.HH",
-"parser.error.topic": "indexing"
-}
-        </value>
+        <name>es_date_format</name>
+        <description>Elasticsearch Date Format</description>
+        <value>yyyy.MM.dd.HH</value>
+        <display-name>Elasticsearch Date Format</display-name>
+    </property>
+    <property>
+        <name>parser_error_topic</name>
+        <description>Parser Error Topic</description>
+        <value>indexing</value>
+        <display-name>Parser Error Topic</display-name>
+    </property>
+    <property>
+        <name>enrichment_workers</name>
```

Comment:
After deploying in Full Dev, I went into Ambari and changed some Enrichment properties like "Enrichment Workers".  After saving the changes, Ambari indicated that I needed to restart all of the Metron-related components.  Even unrelated components like the Metron Management UI.  It only really needed to restart the Enrichment topology. 

I am not sure what logic controls that.  Is that something controlled in this PR?  Or is that external to these changes?